tools/test.py

```diff
@@ -44,11 +44,14 @@
 import errno
 import copy
 import ast
+import resource
 
 from os.path import join, dirname, abspath, basename, isdir, exists
 from datetime import datetime
 from Queue import Queue, Empty
 
+resource.setrlimit(resource.RLIMIT_CORE, (0,0))
```

Comment:
I took a few tentative steps towards using `preexec_fn` in the `subprocess.Popen()` call so that the resource limit only applies when needed, but my Python is not-so-good and I ran into not being sure how to get the necessary configuration indicators passed from the (will have to create it) `AbortTestConfiguration`.

https://stackoverflow.com/a/1689991/436641